vnext/ReactUWP/Base/UwpReactInstance.cpp

```diff
@@ -181,6 +183,11 @@ std::vector<facebook::react::NativeModuleDescription> GetModules(
     []() { return std::make_unique<ClipboardModule>(); },
     messageQueue);
 
+  modules.emplace_back(
+    NativeAnimatedModule::name,
+    [nativeAnimated = std::move(nativeAnimated)]() mutable { return std::make_unique<NativeAnimatedModule>(nativeAnimated); },
```

Comment:
>make_unique [](start = 73, length = 11)

[https://stackoverflow.com/questions/37884728/does-c11-unique-ptr-and-shared-ptr-able-to-convert-to-each-others-type](https://stackoverflow.com/questions/37884728/does-c11-unique-ptr-and-shared-ptr-able-to-convert-to-each-others-type)
In short, you can easily and efficiently convert a std::unique_ptr to std::shared_ptr but you cannot convert std::shared_ptr to std::unique_ptr.